schedule.yaml

```diff
@@ -46,15 +46,33 @@ match_periods:
     description: Virtual league
```

Comment:
What does max_end_time do, particularly in relation to the virtual league? If we have 5 minute slots and with the potential for 29 matches for the virtual league, it would last 145 minutes, which is longer than the max_end_time allows

Reply:
Also have this question for how it works with the other match periods

Reply:
See https://github.com/PeterJCLaw/dummy-comp/blob/main/schedule.yaml#L38-L50 and https://srcomp.readthedocs.io/en/latest/schedule.html which document the behaviour.

Reply:
Thanks. I think the end times are all fine then, except potentially the virtual league one, but that could be changed on the day if needed I guess